docs/rebase.md

```diff
@@ -23,7 +23,7 @@ Commits are shown in order of application, where the base commit (or inferred ro
 
 #### Squash commit with next (`q`)
 
-The selected commit will be squashed with the commit below it.  The two commit messages will be combined into the one below.
+A list of commits will be displayed, the commit under cursor will be squashed with the selected commit.  The two commit messages will be combined into the one.
```

Comment:
I kind of liked the previous 1 key squash. Would it make sense to make this a separate command/shortcut?